immunopepper/io.py

```diff
@@ -0,0 +1,86 @@
+import gzip
+import logging
+import os
+import pickle
+import psutil
+import sys
+
+### intermediate fix to load pickle files stored under previous version
+import spladder.classes.gene as gene
+import spladder.classes.splicegraph as splicegraph
+import spladder.classes.segmentgraph as segmentgraph
+sys.modules['modules.classes.gene'] = gene
+sys.modules['modules.classes.splicegraph'] = splicegraph
+sys.modules['modules.classes.segmentgraph'] = segmentgraph
+### end fix
+
+
+def load_pickled_graph(f):
+    return pickle.load(f)
+
+
+def gz_and_normal_open(file_path,mode='r'):
+    if file_path.endswith('.gz'):
+        mode += 't'
+        file_fp = gzip.open(file_path, mode)
+    else:
+        file_fp = open(file_path, mode)
+
+    return file_fp
+
+
+def print_memory_diags(disable_print=False):
```

Comment:
I find it surprising to find the function here, since it has nothing to do with IO per se. I'd move it to `utils`

Reply:
moved into `utils`